tests/integration/expired_client.go

```diff
@@ -102,7 +102,7 @@ func (s *CCVTestSuite) TestConsumerPacketSendExpiredClient() {
 	delegate(s, delAddr, bondAmt)
 
 	// send CCV packets to consumer
-	s.providerChain.NextBlock()
+	s.nextEpoch()
```

Comment:
I cannot comment directly, but see 6 lines above on line 99. It should also be nextEpoch, right?